client/app/billing/autoRenew/bulk/bulk.html

```diff
@@ -50,6 +50,21 @@
                 </ul>
             </li>
         </ul>
+        <div class="mb-4">
+            <button
+                type="button"
+                class="oui-link"
+                data-ng-click="$ctrl.expand.available = !$ctrl.expand.available"
+                data-ng-bind="($ctrl.expand.available ? 'billing_autorenew_bulk_available_less' : 'billing_autorenew_bulk_available_more') | translate">
```

Comment:
Personal preference:
```suggestion
                data-translate="{{:: $ctrl.expand.available ? 'billing_autorenew_bulk_available_less' : 'billing_autorenew_bulk_available_more' }}">
```